dashboard-for-desktop/articles/dashboard-designer/appearance-customization/coloring.md

```diff
@@ -4,8 +4,51 @@ author: Natalia Kazakova
 legacyId: 17943
 ---
 # Coloring
-The Dashboard Designer provides the capability to manage coloring of dashboard item elements. You can choose whether to use a global color scheme providing consistent colors for identical values across the dashboard or a local color scheme that provides an independent set of colors for each dashboard item. The Dashboard Designer also allows you to edit colors automatically assigned from the default palette.
+The Dashboard Designer allows you to paint dashboard item elements such as chart series points or pie segments.
+
+![Coloring_GlobalColors](../../../images/coloring_globalcolors25370.png)
+
+## Supported Dashboard Items
+You can paint the following [dashboard items](../designing-dashboard-items.md):
+
+* Chart
+* Scatter Chart
+* Pie
+* Pie Map
+* Range Filter
+* Treemap
+
+## Color Scheme Overview
+The dashboard supports two ways to color dashboard item elements:
+
+* A **Global Color Scheme** uses consistent colors for identical values across the dashboard. The image below shows a dashboard that contains Pie and Chart dashboard items. Pie segments and chart series points that correspond to _Wisconsin_, _Oregon_ and _Idaho_ dimension values use identical colors from the default palette.
+	
+	![wdd-coloring-global-scheme](../../../images/wdd-coloring-global-scheme126076.png)
+	
+	 For a global color scheme, the dashboard reserves automatically generated colors for specific values regardless of the filter state.
+* A **Local Color Scheme** uses an independent set of colors for each dashboard item. The image below shows Pie segments that use colors from a local color scheme. These colors do not affect the Chart item that uses a global scheme.
```

Comment:
`These colors do not affect the Chart item that uses a global scheme.`

Is there only one Chart item that uses a global scheme? Or can there be more than one?

Reply:
```suggestion
* A **Local Color Scheme** uses an independent set of colors for each dashboard item. The image below shows Pie segments that use colors from a local color scheme. These colors do not affect the Chart items that use a global scheme.
```